checker/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/checker"
+)
+
+var configFile string
+
+func init(){
+	flag.StringVar(&configFile, "config-file", "", "Config file to use")
+}
+
+func usage(){
+	fmt.Fprintf(os.Stderr, "Usage of %s:\n", os.Args[0])
+	flag.PrintDefaults()
+}
+
+func run() int {
+	if configFile == "" {
+		fmt.Fprintf(os.Stderr, "Error: config-file argument is required\n")
```

Comment:
nit: better to use `CommandLine.Output()` instead of `os.Stderr`, this allows overriding where usage is printed (with `SetOutput`)